packages/ngx-filesize/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ngx-filesize",
-  "version": "3.0.5",
+  "version": "3.0.6",
```

let's revert this, will bump in a separate commit directly in `main`:

```suggestion
  "version": "3.0.5",
```